public/app/features/alerting/unified/components/silences/SilencesTable.tsx

```diff
@@ -29,12 +32,36 @@ export interface SilenceTableItem extends Silence {
 type SilenceTableColumnProps = DynamicTableColumnProps<SilenceTableItem>;
 type SilenceTableItemProps = DynamicTableItemProps<SilenceTableItem>;
 interface Props {
-  silences: Silence[];
-  alertManagerAlerts: AlertmanagerAlert[];
   alertManagerSourceName: string;
 }
 
-const SilencesTable = ({ silences, alertManagerAlerts, alertManagerSourceName }: Props) => {
+const SilencesTable = ({ alertManagerSourceName }: Props) => {
+  const { data: alertManagerAlerts, isLoading: amAlertsIsLoading } =
```

Comment:
```suggestion
  const { data: alertManagerAlerts = [], isLoading: amAlertsIsLoading } =
```
Might be easier than handling undefined later in the code :)